### ISSUE TITLE
Incorrect test units for Beam module

### ISSUE THREAD

**Issue by nopria** [sympy/sympy#20407]
I made a pull request (#20405) which was rejected automatically. Looking at the first of the failed test, as far as I understand, it seems that the test is incorrect. The code of the unit test is at line 1347 of `beam.py`:

```python
from sympy.physics.continuum_mechanics.beam import Beam
from sympy import symbols
R1, R2 = symbols('R1, R2')
b = Beam(8, 200*(10**9), 400*(10**-6))
b.apply_load(5000, 2, -1)
b.apply_load(R1, 0, -1)
b.apply_load(R2, 8, -1)
b.apply_load(10000, 4, 0, end=8)
b.bc_deflection = [(0, 0), (8, 0)]
b.solve_for_reaction_loads(R1, R2)
b.plot_bending_moment()

```

The test result is based on current code, but current code has some sign errors (this is what my PR was trying to fix). Shear at right end of the beam is positive (see image below), whereas it should be negative, given that the positive vertical axis is assumed downward and the reaction at right end is upward.

![image](https://user-images.githubusercontent.com/4848618/98656938-908df180-2341-11eb-9460-1e73621ab31d.png)
 
This is related to discussion in issue #20204, which the above mentioned pull request was addressing.

The question is; how can be fixed the same unit tests supposed to check the pull requests?

**Comment by moorepants** [comment]
You'll need to update the tests and the code in your PR so that everything passes.

**Comment by nopria** [comment]
Ok, I guess this will be quite a long job.
I might try to work on it, but I think it's necessary that someone review and agree on the fix discussed in #20204.

**Comment by moorepants** [comment]
It looks like you got a careful review there and at least one person that agrees. I think making the complete PR is a good next step.
